x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.tsx

```diff
@@ -47,6 +47,28 @@ import { MANAGEMENT_APP_ID } from '../../../common/constants';
 import { PolicyDetailsRouteState } from '../../../../../common/endpoint/types';
 import { WrapperPage } from '../../../../common/components/wrapper_page';
 import { HeaderPage } from '../../../../common/components/header_page';
+import { PolicyAdvanced } from './policy_advanced';
+import { AdvancedPolicySchema } from '../models/advanced_policy_schema';
+
+const AdvancedPolicyForms = React.memo(() => {
+  return (
+    <>
+      {AdvancedPolicySchema.map((advancedField, index) => {
```

Comment:
suggestion to add all of this logic into the `policy_advanced.tsx` file and just import the whole thing into this file.  This would keep all logic for that component contained in one place

Reply:
f44601c417470f867e7aff2f38871e449f948725